src/generic/numberparsing.h

```diff
@@ -217,7 +217,7 @@ really_inline bool is_integer(char c) {
 // We need to check that the character following a zero is valid. This is
 // probably frequent and it is harder than it looks. We are building all of this
 // just to differentiate between 0x1 (invalid), 0,1 (valid) 0e1 (valid)...
-const bool structural_or_whitespace_or_exponent_or_decimal_negated[256] = {
+static const bool structural_or_whitespace_or_exponent_or_decimal_negated[256] = {
```

Comment:
This I can take or leave. It has no bearing either way, since this file is only going to be included once per namespace. const is enough to let the compiler know that the values herein will always be the same as at compile time.

Can it be constexpr? That would tell the compiler what we want it to know more explicitly. I still suspect it doesn't make a difference, but it'd be more accurate, I think?

Reply:
Yes, it should be constexpr.